git-github-introduction.md

```diff
@@ -0,0 +1,413 @@
+# GitHub and Git
+The majority of projects in our lab are hosted on [github.com](https://github.com/).
+GitHub is a web application that can be used for collaboration, code review, and code management.
+It is popularly used in open source projects and by distributed developers.  Contributions to
+our projects are expected to be made using [pull requests](https://help.github.com/articles/using-pull-requests)
+to repositories of their GitHub organization.  For example, the Seattle repositories
+are hosted on the [SeattleTestbed](https://github.com/SeattleTestbed) organization.
+
+Git is a free and open source distributed [VCS](https://en.wikipedia.org/wiki/Version_control_system)
+(Version Control System).  A VCS helps manage changes made to files.  Users can run Git on their local
+computers and upload changes to remote repositories and files hosted on GitHub.  Contributors can edit files
+locally, upload their changes to GitHub, and then initiate a pull request. The following document
+covers the basics of using Git and submitting a pull request to one of our project's GitHub repository.
+
+
+
+
+
+## Git Installation
+[git-scm.com](https://git-scm.com/) provides platform-specific installers for
+[Mac](https://git-scm.com/download/mac),
+[Linux](https://git-scm.com/download/linux), and
+[Windows](https://git-scm.com/download/win) users.
+
+## Configure Git User Information
+
+After installing Git, the next step is for the user to configure his/her author information (username and email address) so that Git will correctly log the author of a commit. 
+
+To configure Git author information, open the Terminal and type `git config
+--global user.name "user's name"`, which will allow you to configure Git on
+your local machine. Each repository you create will use the same user (author)
+configuration if you use the global flag (as in the provided example).
+
+```
+$ git config --global user.name "John Smith"

```

Comment:
I noted in the guide that the `user.name` value is arbitrary, but it is customary to user your full name.  I also decided to retain the user.name configuration since the official Git docs advise users to set it, and the error message one receives when a Git identity is unset asks for a `user.name`.

https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup